src/rgw/rgw_sal_store.h

```diff
@@ -404,9 +404,15 @@ class StoreZone : public Zone {
     virtual ~StoreZone() = default;
 };
 
-class StoreLuaScriptManager : public LuaScriptManager {
+class StoreLuaManager : public LuaManager {
 public:
-  virtual ~StoreLuaScriptManager() = default;
+  virtual ~StoreLuaManager() = default;
+
+  virtual int add_package(const DoutPrefixProvider* dpp, optional_yield y, const std::string& package_name, bool allow_compilation) { return -ENOENT; }
```

Comment:
why do we have stub implementations to the package related functions while the script related functions are pure virtual?

Reply:
do you want them all stubs, or all pure virtual here?